### DISCUSSION TITLE
Import Logins

### DISCUSSION THREAD

**Discussion by gjedeer** [Floorp-Projects/Floorp#880] [Feature Request]
In Firefox in about:logins in the three dots menu there's an Import from a File option.

In Floorp in about:logins this option is missing, making it pretty hard to migrate from Firefox without trusting my passwords to Mozilla Sync, which I would rather not do.

It would be very nice to have the ability to import Firefox passwords to Floorp.

**Comment by vickygamer997** [comment]
Is this in Firefox ESR 115?

**Reply by CutterKnife** [reply]
Yes, this feature is disabled by default in Firefox 115.
Maybe... `signon.management.page.fileImport.enabled` ?

**Reply by gjedeer** [reply]
Yup, that was it - I enabled `signon.management.page.fileImport.enabled` in `about:config` in  Floorp and the option is there now. Thank you.

The Firefox I checked was 122, I didn't think that it would be different in 115 ESR.
